pkg/azuredisk/nodeserver_test.go

```diff
@@ -461,8 +468,10 @@ func TestNodeStageVolume(t *testing.T) {
 		setupFunc     func(*testing.T, FakeDriver)
 		req           csi.NodeStageVolumeRequest
 		expectedErr   error
+		expectedErrV2 error
 		skipOnDarwin  bool
 		skipOnWindows bool
+		skipOnV1      bool
```

Comment:
Rather than add these test cases to the existing combined v1/v2 unit tests, consider create a `nodeserver_v2_test.go` file that contains a separate test function specifically for these test cases. I think you'll be able to avoid adding functions to the `FakeDriver` interface and stubs to `driver` by creating a new test `driverV2` instance through `newFakeDriverV2()` directly.